src/server/routes/api/document/pubmed/searchPubmed.js

```diff
@@ -17,6 +23,18 @@ const DEFAULT_ESEARCH_PARAMS = {
   api_key: NCBI_EUTILS_API_KEY
 };
 
+const safeFetch = url => {
+  const userAgent = `${process.env.npm_package_name}/${process.env.npm_package_version}`;
```

Comment:
Maybe the contact email could be included here.  Otherwise, I guess CR would have to google us if something goes wrong

Reply:
From their [docs](https://github.com/CrossRef/rest-api-doc): 
> Specify a User-Agent header that properly identifies your script or tool and that provides a means of contacting you via email using "mailto:". For example: GroovyBib/1.1 (https://example.org/GroovyBib/; mailto:GroovyBib@example.org) BasedOnFunkyLib/1.4.